app/timetables/tests/test_models.py

```diff
@@ -235,3 +237,49 @@ def test_zero_cycle_day_value_cannot_be_saved(self):
             timetable=self.timetable_object)
 
         self.assertRaises(ValidationError, menu_item_three.save)
+
+
+class EventTest(TestCase):
+    """Tests the Event model."""
+
+    def setUp(self):
+        self.timetable = Timetable.objects.create(
+            name='timtable-item',
+            code='FBI23212',
+            api_key='419223',
+            cycle_length=14,
+            current_cycle_day=2,
+            date_created=datetime.strptime('05 07 2016', '%d %m %Y'),
+            date_modified=datetime.strptime('06 08 2016', '%d %m %Y')
+        )
+        self.future_date = date.today() + timedelta(days=25)
+        self.today_date = date.today()
```

Comment:
@andela-snwuguru you probably will want to mock the date here, or provide a certain date for it than making it rely on current date. you can use an example from the date_created and date_modified fields in the timetable model.